src/components/Organisms/TodoCard/index.jsx

```diff
@@ -42,6 +38,15 @@ const TodoCard = () => {
     }
   };
 
+  useEffect(() => {
+    const data = JSON.parse(localStorage.getItem("taskData"));
+    setTaskList(data);
```

Comment:
![image](https://github.com/OzawaSachio/sachi-frontend-basic-2022-todoapp/assets/83479175/ce293685-80b0-4834-921c-44f6ee5dd37e)

一番最初にこのサイトに訪れたタイミングだとlocalStrageにデータが存在しないので `taskList` に `null` が入ります。
そのため、 `taskElements` を宣言している以下の部分で `null` には `map` メソッドが存在しないよ、と怒られているのが画像の状態です。

```jsx
  const taskElements = taskList.map((task, index) => {
```

```suggestion
    const data = JSON.parse(localStorage.getItem("taskData")) ?? [];
    setTaskList(data);
```
このように [null合体演算子](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) を使うと、localStrageが空であっても対応可能です。

Reply:
挙動を確かめるためには、シークレットウインドウでstorybookを開くのがおすすめです。
（localStrageが空っぽの状態で開くことが）

Reply:
> ![image](https://user-images.githubusercontent.com/83479175/268437114-ce293685-80b0-4834-921c-44f6ee5dd37e.png)
> 
> 一番最初にこのサイトに訪れたタイミングだとlocalStrageにデータが存在しないので `taskList` に `null` が入ります。 そのため、 `taskElements` を宣言している以下の部分で `null` には `map` メソッドが存在しないよ、と怒られているのが画像の状態です。
> 
> ```js
>   const taskElements = taskList.map((task, index) => {
> ```
> 
> このように [null合体演算子](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) を使うと、localStrageが空であっても対応可能です。

「(localStorage.getItem("taskData")) ?? []」と記述した所「SyntaxError: Unexpected token o in JSON at position 1」というエラー文が出てきてきた為、「(localStorage.getItem("taskData") ?? [])」と記述して修正しました。
この修正によって別の不具合が生じていないかレビューしていただけると幸いです。